maker/src/main.rs

```diff
@@ -281,6 +281,12 @@ async fn main() -> Result<()> {
                 tracing::info!(endpoint = %http_endpoint, "HTTP interface is ready");
             })
         }))
+        .attach(AdHoc::on_request("Rocket request", |req, _data| {
+            Box::pin(async move {
+                let request = req;
+                tracing::debug!(%request,  "rocket");
```

Comment:
What does this look like in the logs? Perhaps we can use a slightly more generic log message like "HTTP request? or something? I think we are at the point where the usage of rocket is basically an implementation detail from the outside which I am quite fond of.

Reply:
I don't remember, I think it was something like "rocket Request" . I don't mind changing it to "HTTP" though, so I'll just push a fixup.